internal/logging/logging.go

```diff
@@ -0,0 +1,42 @@
+package logging
+
+import (
+	"io"
+	"log"
+	"os"
+)
+
+var (
+	logFile *os.File
+	logFilePath = "info.log"
```

Comment:
I would recommend use `slog` package that was introduced in 1.21.

1. Structured Logging: slog allows the code to add key-value pairs to your log messages. This can make your logs easier to parse and analyze. For example, you could modify your setOutput function to include additional context information:

2. Log Levels: slog provides functions for different log levels, such as Info, Debug, Warn, and Error. You could use these functions to log messages at different levels of severity.

3. Custom Handlers: slog allows you to define custom handlers for your logs. You could use this feature to customize how your logs are formatted or where they are sent. For example, you could create a handler that sends critical logs to a separate file or a remote logging service.

4. Performance: slog is designed to be fast and efficient. By using slog, you can improve the performance of your logging package.

``` go
import "log/slog"

type SlogLogger struct {
    logger slog.Logger
}

func NewSlogLogger() *SlogLogger {
    return &SlogLogger{
        logger: slog.Default(),
    }
}

func (l *SlogLogger) Info(message string, keyvals ...any) {
    l.logger.Info(message, keyvals...)
}

func (l *SlogLogger) Error(message string, keyvals ...any) {
    l.logger.Error(message, keyvals...)
}
```
https://go.dev/blog/slog

Reply:
In the beginning, I actually thought it might be overkill for the gittuf workflow (as discussed here #54). However, I am currently working on a slog implementation. I am just in the process of writing a custom handler to format key-val pair logs into human readable output to show on terminal.

Reply:
Awesome! Thanks for taking the suggestion. Appreciate that!